crypto/s2n_sequence.c

```diff
@@ -50,8 +51,8 @@ int s2n_sequence_number_to_uint64(struct s2n_blob *sequence_number, uint64_t *ou
     uint8_t shift = 0;
     *output = 0;
 
-    for (int i = sequence_number->size - 1; i >= 0; i--) {
-        *output += ((uint64_t) sequence_number->data[i]) << shift;
+    for (uint32_t i = sequence_number->size; i > 0; i--) {
+        *output += ((uint64_t) sequence_number->data[i-1]) << shift;
```

Comment:
@camshaft should we keep it like that or use `uint64_t`?

Reply:
Same comment as on `s2n_drbg.c`: `int64_t` in place of `uint64_t`.